hack/remote-up-karmada.sh

```diff
@@ -48,9 +48,23 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 "${SCRIPT_ROOT}"/hack/deploy-karmada.sh "${HOST_CLUSTER_KUBECONFIG}" "${HOST_CLUSTER_NAME}" "remote"
 kubectl config use-context karmada-apiserver --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 
+#https://textkool.com/en/ascii-art-generator?hl=default&vl=default&font=DOS%20Rebel&text=KARMADA
+KARMADA_GREETING='
```

Comment:
Can we put this variable into a common file, like in `hack/util.sh`? Thus this can be shared by `hack/remote-up-karmada.sh` and `hack/local-up-karmada`.

Reply:
I agree and will do it.